inc/deprecated/3.6.php

```diff
@@ -1,9 +1,13 @@
-
 <?php
 // phpcs:ignoreFile
 
 defined( 'ABSPATH' ) || exit;
 
+/**
+ * Class aliases.
+ */
+class_alias( '\WP_Rocket\Engine\Optimization\ServiceProvider', '\WP_Rocket\ServiceProvider\Optimization_Subscribers' );
+
```

Comment:
We also need to load the deprecated the class files here too. 

Why? Composer won't autoload them. Why? Their namespace will not match their new `inc/deprecated/` filesystem location (and we don't want to change their namespace).

Reply:
@hellofromtonya Having these classes in [composer’s classmap](https://github.com/wp-media/wp-rocket/pull/2474/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R63) seems to be enough, they are autoloaded:
```
	"autoload": {
		"classmap": [
			"inc/classes",
			"inc/deprecated",
			"inc/vendors/classes"
		],
```

Reply:
Let's do don't add it to Composer's autoload. Why? In the future, we want to add a way to disable loading deprecated files, i.e. to lower memory consumption when a website doesn't need the older stuff.

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2474/commits/ce06cb0a809f7d52c4ede8ca27a6040666319801.